### PR TITLE
Create version-sweeper workflow YML file

### PR DESCRIPTION
**Pull request by IEvangelist**
## Summary

Create version-sweeper workflow YML file to expose .NET version sweeper features.

- Automated issue creation when *proj/*sln files are targeting non-LTS (or current) versions, labeled by:
  - [`dotnet-target-version`](https://github.com/dotnet/docs/issues?q=is%3Aopen+is%3Aissue+label%3Adotnet-target-version)
- This is configurable and the configurations bits are detailed here: https://github.com/dotnet/versionsweeper#configure-action
- There is an existing _ignore_ file here: https://github.com/dotnet/docs/blob/master/dotnet-versionsweeper.json
- This action can also be manually ran through the Actions tab

Relates to #22271

### CODE REVIEW

**Review by BillWagner — APPROVED** [review]
This is great @IEvangelist 

I'm ready to :shipit:

note: we should add this to the samples and api-docs repos as well.

File: .github/workflows/version-sweep.yml [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+      # Runs a single command using the runners shell
+      - name: 'Print manual run reason'
+        if: ${{ github.event_name == 'workflow_dispatch' }}
+        run: |
+          echo 'Reason: ${{ github.event.inputs.reason }}'
+
+      # Start the .NET version sweeper, scan projects/slns for non-LTS (or currrent) versions
+      - name: .NET version sweeper
+        id: dotnet-version-sweeper
+        uses: dotnet/versionsweeper@v1.0
```
Comment (IEvangelist) [inline .github/workflows/version-sweep.yml:38]:
```suggestion
        uses: dotnet/versionsweeper@v1.2
```

**Comment by IEvangelist** [comment]
> This is great @IEvangelist David Pine FTE
> 
> I'm ready to :shipit:
> 
> note: we should add this to the samples and api-docs repos as well.

I've created the two corresponding PRs:

- https://github.com/dotnet/samples/pull/4376
- https://github.com/dotnet/dotnet-api-docs/pull/5371

**Comment by adegeo** [comment]
@IEvangelist cool! Can you hit the dotnet/docs-desktop repo too?

**Comment by IEvangelist** [comment]
> @IEvangelist David Pine FTE cool! Can you hit the dotnet/docs-desktop repo too?

Done: https://github.com/dotnet/docs-desktop/pull/211

**Pull request merged**